src/FolderSync.cpp

```diff
@@ -473,16 +473,28 @@ void CFolderSync::SyncFile(const std::wstring& path, const PairData& pt)
             fd.ft = fDataOrig.ftLastWriteTime;
             if (bCopyOnly)
             {
+                bool bCopyFileResult;
                 CCircularLog::Instance()(_T("INFO:    copy file %s to %s"), orig.c_str(), crypt.c_str());
```

Comment:
just use
bool bCopyFileResult = CopyFile(...
instead of first declaring the variable and then assigning a value a line later.

Reply:
Done